chatcli_gpt/chatcli.py

```diff
@@ -363,6 +364,9 @@ def answer(request_messages, model, plugins, stream=True, tags=None):
         plugins=plugins,
     )
 
+    # TODO:
+    # - [ ] wrote more plugins and see what's good interface for plugins
+    # - [ ] middleware pattern for plugins
```

Comment:
💯 This was my plan here. I don't know if it ends up looking like middleware, but my general approach was add a few plugins and then figure out the interface.

Reply:
Sorry, let me remove these notes; didn't mean for a proposal, more random notes when I was reading the code.

Reply:
I'm happy to have these. This is the plan.

Reply:
Added back.